app/javascript/app/components/country-climate-vulnerability/country-climate-vulnerability-selectors.js

```diff
@@ -0,0 +1,83 @@
+import { createSelector } from 'reselect';
+import isEmpty from 'lodash/isEmpty';
+
+const getIso = state => state.iso || null;
+const getAdaptations = state => state.adaptations || null;
+
+const getSection = (sections, slug) => sections.find(s => s.slug === slug);
+
+const getData = (sections, iso, slug) => {
+  const section = getSection(sections, slug);
+  return (
+    section &&
+    section.values &&
+    section.values.find(country => country.location === iso)
+  );
+};
+
+const getListData = (sections, iso, slug) => {
+  const items = sections.filter(section => section.slug.startsWith(slug));
+  const itemList = [];
+  if (
+    !items.some(item => item.values.some(country => country.location === iso))
+  ) {
+    itemList.push('No data');
```

Comment:
Why don't return the empty array directly?

Reply:
It is maybe different having no vulnerabilities or having no data about a country. We will have to check the final data.